.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -19,6 +19,10 @@ parameters:
     type: string
     default: ""
 
+  - name: LOCK_WAIT_TIMEOUT_SECONDS
+    type: string
```

Comment:
Why string not number?

Reply:
In Azure pipeline, actually, all things are string.
Besides, with string, we can default to pass a blank